infrastructure/json/src/main/java/tech/pegasys/teku/infrastructure/json/types/CoreTypes.java

```diff
@@ -106,6 +107,15 @@ public class CoreTypes {
               HttpErrorResponse.Builder::message)
           .build();
 
+  public static final DeserializableTypeDefinition<Eth1Address> ETH1ADDRESS_TYPE =
```

Comment:
Ah, I see why the move happened. This doesn't belong in CoreTypes (I'm not sure `HttpErrorResponse` should be in here either to be fair...). We have `EthereumTypes` which would be a good place for it, but currently that sits in the beacon rest API module and really should be moved to something shared between the beacon rest API and validator rest API - quite likely a `ethereum:json-types` module.